packages/services/src/infrastructure/api/hiro/hiro-stacks-api.client.ts

```diff
@@ -52,11 +52,20 @@ export type HiroNftHolding = NonFungibleTokenHolding;
 
 @injectable()
 export class HiroStacksApiClient {
+  private readonly _axios: AxiosInstance;
+
   constructor(
     @inject(Types.CacheService) private readonly cache: HttpCacheService,
     @inject(Types.SettingsService) private readonly settings: SettingsService,
     private readonly limiter: RateLimiterService
-  ) {}
+  ) {
```

Comment:
Nice! Thanks for adding this 💪  maybe this negates the need for increased limits elsewhere?